src/App.tsx

```diff
@@ -1,12 +1,26 @@
 import React from 'react';
 import './App.scss';
 
-// import todos from './api/todos';
-// import users from './api/users';
+import toDos from './api/todos';
+import users from './api/users';
+
+import { ToDoList } from './components/ToDoList/ToDoList';
+
+const preparedList = toDos.map((toDo: ToDo) => (
+  {
+    ...toDo,
+    user: users.find(user => (
+      user.id === toDo.userId
+    )) || null,
+  }
+));
 
 const App: React.FC = () => (
   <div className="App">
-    <h1>Static list of todos</h1>
+    <h1>List Of tasks:</h1>
+    <ToDoList
+      preparedList={preparedList}
+    />
```

Comment:
you don't need to wrap only one prop
```suggestion
    <ToDoList list={preparedList}/>
```